### DISCUSSION TITLE
Github pages PWA isn't working

### DISCUSSION THREAD

**Discussion by melonmasteristaken** [community/community#118751] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

So I was trying to make my site a PWA, I’ve done things like this before with help from guides.

However this time, im not sure what happened and it seems like it just doesn't work no matter what i do
(If it helps, PWABuilder detected nothing wrong with my site)

Repo: https://github.com/1d10t1c-stud10s/1d10t1c-stud10s.github.io
Website link: https://1d10t1c-stud10s.github.io

**Comment by dorathoto** [comment]
has several errors:
check what is required for PWA and test it on your website.

Ex:
The manifest.json file is missing

**Reply by melonmasteristaken** [reply]
Yeah I used site.webmanifest, it is the same as a manifest (right?).

**Reply by melonmasteristaken** [reply]
Could it be because I used the root of the site?

Eg.
example.com/

**Reply by LiteBrite82** [reply]
From what I can see, it looks like you just need to link the manifest in your index.html file: https://developer.mozilla.org/en-US/docs/Web/Manifest

**Reply by melonmasteristaken** [reply]
I did try that, however it still didn't work

You can check my index.html, it's all there

**Comment by melonmasteristaken** [comment, marked as answer]
Welp, i redid the website a few months later and the manifest just worked. Must've been a conflict with the site theme

**Reply by LiteBrite82** [reply]
Nice!
